Week2/todo_app/program.js

```diff
@@ -22,38 +22,86 @@ class TodoModel {
     }
 
     create(description, callback) {
-        // Write code and query to create a new TODO item
+        const sqlInsert = {
+            text: description,
+            is_completed: 0,
+            user_id: 1
```

Comment:
Very nice work to use this formatted input! You can leave `user_id` as `null` if it's not specified though :)